### PR TITLE
[stable/tomcat] Update api versions to support k8s 1.16

### PR DESCRIPTION
**Pull request by antongorkovenko**
Signed-off-by: Anton Gorkovenko <gostom@gmail.com>

#### Is this a new chart
no
#### What this PR does / why we need it:
Update api versions (deprecated -> actual) to support k8s 1.16
ref: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/
#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by antongorkovenko** [comment]
@mattfarina @zanhsieh please approve

**Comment by zanhsieh** [comment]
@antongorkovenko 
Pinged @jlegrone already. Sorry this chart has no owner.

File: stable/tomcat/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 name: tomcat
 description: Deploy a basic tomcat application server with sidecar as web archive container
 icon: http://tomcat.apache.org/res/images/tomcat.png
-version: 0.3.0
-appVersion: "7.0"
+version: 0.4.0
+appVersion: "9.0"
```
Comment (jlegrone) [inline stable/tomcat/Chart.yaml]: This doesn't appear to be related and isn't mentioned in the PR description. Is the current appVersion incorrect?

Comment (antongorkovenko) [reply stable/tomcat/Chart.yaml]: reverted

**Comment by antongorkovenko** [comment]
@jlegrone please review

**Comment by zanhsieh** [comment]
@antongorkovenko 
I would suggest you ping @jlegrone on kubernetes slack channel (kuberenetes.slack.com; search his name on top of slack app and start direct message).

**Review by jlegrone — APPROVED** [review]
Thanks!

/lgtm
